src/Config/Auth.php

```diff
@@ -185,6 +188,28 @@ class Auth extends BaseConfig
         'rememberLength'     => 30 * DAY,
     ];
 
+    /**
+     * --------------------------------------------------------------------
+     * JWT Authenticator Configuration
+     * --------------------------------------------------------------------
+     * These settings only apply if you are using the JWT Authenticator
+     * for authentication.
+     *
+     * - secretKey   The secret key. Needs more than 256 bits random string.
+     *               E.g., $ php -r 'echo base64_encode(random_bytes(32));'
+     * - algorithm   JWT Signing Algorithms.
+     * - timeToLive  Specifies the amount of time, in seconds, that a token is valid.
+     *
+     * @var array<string, bool|int|string>
+     */
+    public array $jwtConfig = [
+        'issuer'     => '<Issuer of the JWT>',
+        'audience'   => '<Audience of the JWT>',
```

Comment:
According to the chart of supported PHP libs on the JWT site, Firebase's implementation doesn't support audience checks. If that's the case do we need a setting for it?

Reply:
I'm not familiar with "audience", probably a symptom of only ever using Firebase's implementation. 😂

Reply:
It is a Registered Claim.
> [Registered claims](https://tools.ietf.org/html/rfc7519#section-4.1): These are a set of predefined claims which are not mandatory but recommended, to provide a set of useful, interoperable claims. Some of them are: iss (issuer), exp (expiration time), sub (subject), aud (audience), and [others](https://tools.ietf.org/html/rfc7519#section-4.1).
https://jwt.io/introduction

I'm going to change the property so that users can remove it.

```php
    public array $jwtConfig = [
        'payload' => [
            'iss' => '<Issuer of the JWT>',
            'aud' => '<Audience of the JWT>',
        ],
        // ...
    ];
```

Reply:
I changed `$jwtConfig`.
https://github.com/codeigniter4/shield/pull/195/files#diff-c48af47762f5a9ba44e0e0b95dd538d48276cf1bc59a8c157bdcddc812184d54R208